benchmark/v8/get-stats.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common.js');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  method: [
+    'getHeapStatistics',
+    'getHeapSpaceStatistics'
+  ],
+  n: [1e6],
+  flags: ['--ignition --turbo', '']
```

Comment:
Yes, the `flags` in the `options` is applied to all runs...to compare the performance under different flags I would suggest something like

```js
const bench = common.createBenchmark(main, {
  method: [
    'getHeapStatistics',
    'getHeapSpaceStatistics'
  ],
  n: [1e6],
  pipeline: ['new', 'old']
});

function main(conf) {
  if (conf.pipeline === 'new') {
    v8.setFlagsFromString('--ignition');
    v8.setFlagsFromString('--turbo')
  }
}
```

Unless we have to turn on the flag during bootstrapping..in that case we do need to add special support in the benchmark suite.